.changes/next-release/feature-AWSCRTbasedS3Client-353870f.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "feature",
+    "category": "AWS CRT-based S3 Client",
```

Comment:
Technically affects other CRT related components like signing right? Might be better just as "AWS SDK for Java v2"

Reply:
I looked at other entries in the changelog, but perhaps we're not consistent - not surprising.

Reply:
Hmm, I thought it only affects s3 client, is that not the case?

Reply:
oh nvm, this is version bump, yeah, should be `AWS SDK for Java v2`